modules/golang/golang

```diff
@@ -1,5 +1,11 @@
 compdef _golang go
 
+function _deps(){
+	app="go"
+
+	$(hash $app 2>/dev/null) || $(myzsh error "Couldn't find application ${app}" && return 1)
```

Comment:
The `return` inside the subshell doesn't return from the function like you expect. It returns from the subshell.